docs/assembly-control-unit.md

```diff
@@ -171,47 +281,105 @@ Heat  |  Stir |
 
-- Begin by stripping the ends of all wires and cables. These will be inserted into various screw terminals on the relays and Phoenix connector. When working with screw terminals, remember that they are only as secure as they are tight. Use firm but gentle turns when screwing down terminals - you want them to be tight, but you don't want to damage the slots on the heads of the screws. You should be able to lightly tug on the wires without them moving.
 - Wiring the 12v half of the sub-assembly
-  - Insert the 3 short wires into the (+) terminals of the 8-pin connector and tighten the terminals (See ASCII art above). Later, we will put the switched lead from the button into the fourth (+) terminal.
-  - Insert one of each of the wires from the 4-wire cable into the open (-) terminals and screw them down.
-  - Insert the other ends of that cable into one of the 3-wide lever connectors. Each slot on the connector should accomodate 2 of the wires. Twisting the wires together with pliers may help. In the unused slot, insert one of the black wires - later we will connect it to the (-) side of the 1-to-3 lever connector that will act as the 12v power manifold.
-  - Insert each of the short wires into the controlled side of the relay in a terminal labeled "NO" (normally open). Screw the terminals down gently but firmly.
-  - Insert the corresponding ends of the 3-wire cable into the terminals labeled "COM" (common) and tighten the terminals.
-  - Insert the other ends of the 3-wire cable into the other 3-wide lever connector, again doubling up in one of the slots. In the remaining open slot, insert one of the short red wires, which will later connect to the 12v manifold.
+-Wiring the Phoneix connector
+  - Positives. On the back on the female 8-pin Phoneix connector, insert a 4" green, red, blue wire into the correct positive (+) terminal and tighten the terminals. The black terminal will be empty for now.
+      - The wire pattern will be reversed when adding wires from the the back of the female connector. So from left to right, skip the first two terminals and then place the green +, red + and blue + wires.
+
+      <IMG ALT="Cable order" SRC="./media/control-unit/wire_rev.png" WIDTH="200" />
+
+Note: When working with screw terminals, remember that they are only as secure as they are tight. Use firm but gentle turns when screwing down terminals - you want them to be tight, but you don't want to damage the slots on the heads of the screws. You should be able to lightly tug on the wires without them moving.
+
+  - Negatives. On the Phoenix connector, take the 6" 4-wire cable and insert each of its wires (black, green, red, blue) into the correct negative (-) terminal and screw them down
+  - Take the other end of the 4-wire cable and place the black and green wire in their own slots of the 4-to-1 wire connector. Then put the red and blue wires together in one slot. That leaves the last slot open for power (-12V), put the 4" black wire in that slot.
```

Comment:
I think we want to call this a "3-lever wire connector" or something similar. 4-to-1 is confusing to me.

Maybe a sub-bullet here:
- Note: depending on the thickness of the wire insulation, the wire connector may not be necessary. You may be able to plug all (-) wires directly into the (-) side of the 12v manifold.